p2p/discover/v5_udp_test.go

```diff
@@ -82,13 +79,7 @@ func startLocalhostV5(t *testing.T, cfg Config) *UDPv5 {
 
 	// Prefix logs with node ID.
 	lprefix := fmt.Sprintf("(%s)", ln.ID().TerminalString())
-	cfg.Log = testlog.LoggerWithHandler(t, log.FuncHandler(func(_ context.Context, r slog.Record) error {
-		if r.Level <= log.LevelTrace {
-			return nil
-		}
-		t.Logf("%s %s", lprefix, log.TerminalFormat(r, false))
-		return nil
-	}))
+	cfg.Log = testlog.Logger(t, log.LevelTrace).With("node-id", lprefix)
```

Comment:
This change is necessary because `TerminalFormat` is now a private member method of `terminalHandler`.

It has the downside of making it harder to read the output of the tests because the `node-id` is not the leftmost item on the output lines.